local/bin/py/build/pull_rbac.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import sys
+import requests
+import json
+from os import getenv
+
+def pull_rbac():
+  api_endpoint = 'https://app.datadoghq.com/api/v2/permissions'
+  headers = {'DD-API-KEY': sys.argv[1], 'DD-APPLICATION-KEY': sys.argv[2]}
+  formatted_permissions_dict = {}
+
+  r = requests.get(api_endpoint, headers=headers)
+
+  if r.status_code == requests.codes.ok:
+    json_result = r.json()
+    data = json_result.get('data', []);
+
+    for permission in data:
+      group_name = permission['attributes']['group_name']
+      permission_name = permission['attributes']['name']
+
+      # Remove legacy logs permissions from dictionary before converting to JSON.  These legacy permissions are hard-coded in rbac-permissions-table partial until they can be deprecated.
+      if permission_name in ('logs_live_tail', 'logs_read_index_data'):
+        del permission
+      else:
+        if group_name not in formatted_permissions_dict.keys():
+          formatted_permissions_dict[group_name] = [permission]
+        else:
+          formatted_permissions_dict[group_name].append(permission)
+
+    formatted_permissions_json = json.dumps(formatted_permissions_dict)
+
+    with open('data/permissions.json', 'w+') as outfile:
+      outfile.write(formatted_permissions_json)
+  else:
+    print('RBAC api request failed.')
```

Comment:
if you print `r` here it may help users of the script better understand what went rong

Reply:
done - thank you!